tools/api-markdown-documenter/src/RenderHtml.ts

```diff
@@ -66,14 +50,11 @@ export interface RenderDocumentsAsHtmlOptions
  *
  * @param documents - The documents to render. Each will be rendered to its own file on disk per
  * {@link DocumentNode.documentPath} (relative to the provided output directory).
- * @param renderConfig - Configuration for rendering {@link DocumentNode}s as HTML.
- * @param fileSystemConfig - Configuration for writing document files to disk.
- * @param logger - Receiver of system log data. Default: {@link defaultConsoleLogger}.
  *
  * @alpha
  */
 export async function renderDocumentsAsHtml(
-	documents: DocumentNode[],
+	documents: readonly DocumentNode[],
```

Comment:
Nice! I didn't know this could be done in arguments to a free function. And experimenting in TSPlaygound makes it clear this makes array-modifying functions unavailable, and even trying to pass it to a function that takes a mutable array is a typecheck failure.